config/locales/en.yml

```
@@ -799,7 +799,7 @@ en:
   account_price_group_members:
     new:
       label:
-        search_term: "Search by payment source"
+        search_term: "Search by payment source number, accounts receivable number, or description, or payment source owner's name, NetID, or username"
```

```suggestion
        search_term: "Search by payment source number, description, accounts receivable number, or payment source owner's name, NetID, or username"
```
Let's update the search_term label on line 167 as well.